atlp-rwanda/champs-ec-be
# #187300235 Verify user email after registration

#### What does this PR do?

#### Description of Task to be completed?

- Setting Up mailer
- Setting mailer templates
- Send token and URI for verification after registration
- Verify User email


#### How should this be manually tested?
- In normal registration api URL, [baseUrl/api/users/signup]
- Create account
- Check email for verification
- Click verification link in email
